Module not found: Can't resolve 'ably' after upgrading to 2.0.3

I upgraded from 1.2.49 to 2.0.3 and the next time I started my react project, It gave me this error:

`Module not found: Can't resolve 'ably' in '...'`

┆Issue is synchronized with this [Jira Task](https://ably.atlassian.net/browse/ECO-4776) by [Unito](https://www.unito.io)

Hi @Mosquid !
Unfortunately, I couldn't reproduce the `Module not found: Can't resolve 'ably' in '...'` error following your steps with our [ably-js nextjs example repo](https://github.com/ably-labs/ably-nextjs-fundamentals-kit) (specifically based on [this commit](https://github.com/ably-labs/ably-nextjs-fundamentals-kit/commit/c5c7965c043b4fee77a24b0a2357adb70d228940) as the latest commit in repo is already using ably-js v2).

After upgrading to ably-js v2 I only got expected errors, such as `Package path ./promises is not exported from package` and `TypeError: ably__WEBPACK_IMPORTED_MODULE_1__.Realtime.Promise is not a constructor`, due to changes to our exports configuration in ably-js v2. Required changes you need to do in the codebase when upgrading to ably-js v2 are covered in our [v2 migration guide](https://github.com/ably/ably-js/blob/main/docs/migration-guides/v2/lib.md), and [react-hooks v2 migration guide](https://github.com/ably/ably-js/blob/main/docs/migration-guides/v2/react-hooks.md).

If you are still experiencing this problem, could you please share more information about your project setup that might affect dependency resolution (like your `tsconfig.json`) and code example of how are you trying to import and use Ably client and react hooks? Thank you!

Hi @VeskeR,
No, unfortunately not. Here is the code of my ably provider. 

```
import { AblyProvider } from "ably/react";
import * as Ably from "ably";
import { FC } from "react";
import { observer } from "mobx-react";
import { ErrorInfo } from "ably";

const client = new Ably.Realtime({
  httpMaxRetryCount: 5,
  authCallback: async (_, callback) => {
    try {
      // custom auth
    } catch (error) {
      callback(error as ErrorInfo, null);
    }
  },
  authParams: {
    path: location.pathname,
  },
});

const AblyClientProvider: FC = ({ children }) => {
  return <AblyProvider client={client}>{children}</AblyProvider>;
};

export default observer(AblyClientProvider);
```

`Error: Module not found: Can't resolve 'ably' in AblyClientProvider.tsx`

--- a/package.json
+++ b/package.json
@@ -7,7 +7,10 @@
     "url": "https://github.com/ably/ably-js/issues",
     "email": "support@ably.com"
   },
+  "main": "./build/ably-node.js",
+  "browser": "./build/ably.js",
   "react-native": "./build/ably-reactnative.js",
+  "typings": "./ably.d.ts",
   "exports": {
     ".": {
       "types": "./ably.d.ts",
@@ -24,7 +27,6 @@
       "import": "./react/mjs/index.js"
     }
   },
-  "typings": "./ably.d.ts",
   "files": [
     "build/**",
     "ably.d.ts",

